### PR TITLE
fix(refs T34963): Add existing Attrs to Link Node

### PR DESCRIPTION
**Pull request by salisdemos**
Add existing Attributes to Link Node
Not doing this caused a
`cannot read property of undefined (reading forEach)`
 And it should not harm, even more prevent unwanted side effects

After the fix, the FE-error disappears, but the described Bug still remains, because something in the BE happens, that strippes additional attributes next to href. :-/ (see comment in Ticket)

**https://yaits.demos-deutschland.de/T34963**

### CODE REVIEW

File: src/components/core/DpEditor/DpEditor.vue [hunk 1]
```diff
@@ -128,7 +128,7 @@
                 @keydown.tab.shift.exact="toggleSubMenu('diffMenu', false)"
                 :disabled="readonly">
                 <dp-icon
-                  class="align-text-top"
+                  class="align-text-top inline-block"
```
Comment (salisdemos) [inline src/components/core/DpEditor/DpEditor.vue:131]: this was supposed to be another PR wich fixes T34973

File: src/components/core/DpEditor/libs/editorCustomLink.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ export default Link.extend({
 
   addOptions () {
     return {
+      ...this.parent?.(),
```
[the remark this answers is not in the document]
Comment (salisdemos) [reply src/components/core/DpEditor/libs/editorCustomLink.js:9]: as far as I understood, it calls an anonymous function from parent.
 (But I just copied it from the documentation https://tiptap.dev/guide/custom-extensions/#settings)

[the remark this answers is not in the document]
Comment (spiess-demos) [reply src/components/core/DpEditor/libs/editorCustomLink.js:9]: Diving deeper into it, i see that `this.parent` accesses the [`parent` property of the Mark class](https://github.com/ueberdosis/tiptap/blob/develop/packages/core/src/Mark.ts#L417), and the `.?()` leads to all properties of  `this.parent` being spreaded into the object if `this.parent` is a function. This is a good example for code being too clever to be understood. 

I'd suggest to add a comment here with a link to the docs you mentioned.

[the remark this answers is not in the document]
Comment (salisdemos) [reply src/components/core/DpEditor/libs/editorCustomLink.js]: good idea 4cbb7d84d4546c24c32172f6e3821ac44786a733
